admin/links/class-link-query.php

```diff
@@ -119,11 +123,13 @@ public static function get_unprocessed_count( array $post_types ) {
 
 		// @codingStandardsIgnoreStart
 		$query = '
-			SELECT COUNT( ID )
-			  FROM ' . $wpdb->posts . ' 
-			 WHERE post_status = "publish" 
-			   AND post_type IN ( ' . $post_types . ' ) 
-			   AND ID NOT IN ( SELECT object_id FROM ' . $count_table . ' )';
+			SELECT COUNT( posts.ID )
+			  FROM ' . $wpdb->posts . ' AS posts
+		 LEFT JOIN ' . $count_table . ' AS yoast_meta
+				ON yoast_meta.object_id = posts.ID
+			 WHERE posts.post_status = "publish" 
+			   AND posts.post_type IN ( ' . $post_types . ' ) 
```

Comment:
Lines 130 and 131 have extra trailing spaces (many editors will remove them producing a changed file)